mappings/net/minecraft/entity/mob/ZombifiedPiglinEntity.mapping

```diff
@@ -10,6 +10,7 @@ CLASS net/minecraft/class_1590 net/minecraft/entity/mob/ZombiePigmanEntity
 		ARG 2 spawnType
 		ARG 3 pos
 		ARG 4 random
+	METHOD method_20806 getNewAngerTime ()I
```

Comment:
Is this
 - the time until it will get angry,
 - the time it will get angry at,
 - the time it has been angry for, or
 - the time it will be angry for?

I think the name should make this clear.

Reply:
The time it will be angry for, hence the `New`.

Reply:
Then I think `getNewAngerDuration` would be better.

Reply:
Or `chooseAngerDuration`

Reply:
```suggestion
	METHOD method_20806 getNewAngerDuration ()I
```